.claude/scripts/watch-empty-queue.sh

```
@@ -0,0 +1,65 @@
+}
+trap cleanup EXIT
+
+# シグナルハンドリング
+shutdown() {
+  echo ""
+  echo "Shutting down..."
+  exit 0
+}
+trap shutdown SIGINT
+
+ME=$(gh api user -q '.login')
```

Shellcheck (0.11.0)

[warning] ME appears unused. Verify use (or export if used externally).

(SC2034)